crypto/asn1/asn1_err.c

```diff
@@ -1,6 +1,6 @@
 /*
  * Generated by util/mkerr.pl DO NOT EDIT
- * Copyright 1995-2018 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 1995-2019 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
It seems that in almost all the `*_err.c` files, only the copyright line was changed.  Would you mind undoing those?

Reply:
These changes are automatic, by `mkerr.pl`.

Reply:
Yeah well, mkerr doesn't compare generated files with the previous version (if there is one), so this demands human scrutiny.

We might want to add a content diff check, but there are other mechanisms in place to avoid generating new files unnecessarily, so this case where only the copyright year is changed is unusual.

Reply:
I've isolated the improvements to `mkerr.pl` and just forced a rebuild of `crmferr.h` to test and demonstrate their effect (which also updated/corrected the License line of that header file).

@levitte, ok this way?